### ISSUE TITLE
Error with Phoenix 1.1.2

### ISSUE THREAD

**Issue by seantanly** [gjaldon/heroku-buildpack-phoenix-static#15]
With the latest version of Phoenix 1.1.2, seem that there are major changes to the way the default Phoenix js/css files are included. [CHANGELOG](https://github.com/phoenixframework/phoenix/blob/master/CHANGELOG.md) [UPGRADE INSTRUCTIONS](https://gist.github.com/chrismccord/d5bc5f8e38c8f76cad33)

I'm getting the following errors even though I have set custom settings for Node, NPM and compile commands. Unfortunately, I'm not too familiar with how the static assets build process works to give more info what went wrong or how to fix it.

`phoenix_static_buildpack.config`
```
node_version=5.4.0
npm_version=3.3.12
config_vars_to_export=(DATABASE_URL)
compile="phoenix_static_buildpack.compile"
```
`phoenix_static_buildpack.compile`
```
info "Building Phoenix static assets"
brunch build --production
mkdir -p priv/static
mix phoenix.digest
```

Errors from console,
```
-----> Loading configuration and environment
       Loading config...
       Will use the following versions:
       * Node 5.4.0
       Will export the following config vars:
       * Config vars DATABASE_URL
       * MIX_ENV=prod

-----> Installing binaries
       Downloading node 5.4.0...
       Installing node 5.4.0...
       Using default npm version

-----> Building dependencies
       Installing and caching node modules
       npm ERR! addLocal Could not install /cache/deps/phoenix
       npm ERR! addLocal Could not install /cache/deps/phoenix_html
       npm ERR! Linux 3.13.0-71-generic
       npm ERR! argv "/tmp/build/.heroku/node/bin/node" "/tmp/build/.heroku/node/bin/npm" "install" "--quiet" "--unsafe-perm" "--userconfig" "/tmp/build/npmrc"
       npm ERR! node v5.4.0
       npm ERR! npm  v3.3.12
       npm ERR! path /cache/deps/phoenix
       npm ERR! code ENOENT
       npm ERR! errno -2
       npm ERR! syscall open

       npm ERR! enoent ENOENT: no such file or directory, open '/cache/deps/phoenix'
       npm ERR! enoent This is most likely not a problem with npm itself
       npm ERR! enoent and is related to npm not being able to find a file.
       npm ERR! enoent

       npm ERR! Please include the following file with any support request:
       npm ERR!     /cache/npm-debug.log
       npm WARN package.json @ No description
       npm WARN package.json @ No README data
       npm WARN package.json @ No license field.
remote: cp: cannot stat ‘node_modules’: No such file or directory
```

**Comment by seantanly** [comment]
@chrismccord https://github.com/gjaldon/heroku-buildpack-phoenix-static/pull/16 has came to the rescue. :)

**Comment by seantanly** [comment]
@gjaldon @chrismccord @HashNuke Thanks guys for working on this. Things are working fine now. :)

**Comment by gjaldon** [comment]
Thanks for reporting the issue, Sean!

On Sun, Jan 10, 2016 at 1:42 PM, Sean Tan <notifications@github.com> wrote:

> Closed #15
> <https://github.com/gjaldon/heroku-buildpack-phoenix-static/issues/15>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gjaldon/heroku-buildpack-phoenix-static/issues/15#event-510380841>
> .
>



-- 
Gabe Jaldon
